pkg/genericapiserver/config.go

```diff
@@ -337,18 +346,64 @@ func (c Config) New() (*GenericAPIServer, error) {
 		})
 	}
 
+	if len(c.AuditLogPath) != 0 {
```

Comment:
Let's add to our running issue.  I think all the audit stuff should be gathered up into a pointer to a struct.